ra/saphana-controller-common-lib

```diff
@@ -316,6 +320,8 @@ function saphana_init_scoring_tables() {
        "[234]   P master[123]          .*       140             'primary - running, configured master nameserver role (failover candidate)'"
        "[015-9] P master[123]          .*        90             'primary - not running, but configured master nameserver role '"
        "[0-9]   P [^:]*:slave          .*        60             'primary - any running status, slave master nameserver role (should not happen for ScaleUp)'"
+       "0       P [^:]*:[?:-]          .*      NULL             'primary - lss=0 defect, keep scoring'"
```

Comment:
Just for curiosity - is this on the right position inside the table? The 2 rows above also will catch a value of '0'